### PR TITLE
Fresh start

### PR DESCRIPTION
**Pull request by alex0112**
Wiped away everything and regenerated the app.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,11 +7,11 @@ The journey begins.
 
 ## Description
 
-Kepler is a stargazing tool—not for those lovely celestial orbs of light, but for your stars you make on GitHub and GitLab.
+Kepler is a stargazing tool—not for those lovely celestial orbs of light, but for your stars on GitHub, GitLab, and beyond.
 
-GitHub stars are great—they let you bookmark interesting projects to view later. What is *not* great is GitHub's interface for managing your stars. If you're anything like [alex0112](https://github.com/alex0112), sorting through your 1k+ stars is, well, impossible on GitHub. This project aims to be a better way to manage your stars so you can keep up on all your favorite projects.
+GitHub stars are great-- they let you bookmark interesting projects to view later. What is *not* great is GitHub's interface for managing your stars. If you're anything like [alex0112](https://github.com/alex0112), sorting through your 1k+ stars is, well, impossible on GitHub. This project aims to be a better way to manage your stars so you can keep up on all your favorite projects.
 
-*More details to come…*
```
Comment (ashton314) [inline README.md:14]: Typographic nit picks: revert `--` back to an m-dash `—`, and change the `...` to a proper ellipsis `…`.

File: README.md [hunk 2]
```diff
@@ -7,11 +7,11 @@ The journey begins.
 
 ## Description
 
-Kepler is a stargazing tool—not for those lovely celestial orbs of light, but for your stars you make on GitHub and GitLab.
```
Comment (ashton314) [inline README.md:10]: Great edits here

File: LICENSE [hunk 3]
```diff
@@ -1,21 +0,0 @@
-MIT License
```
Comment (ashton314) [inline LICENSE:1]: Suggestion: put the MIT license back, unless you want to re-license under something else.

File: assets/webpack.config.js [hunk 4]
```diff
@@ -1,51 +0,0 @@
-const path = require('path');
```
Comment (ashton314) [inline assets/webpack.config.js:1]: Good. Burn webpack with fire.

File: config/config.exs [hunk 5]
```diff
@@ -1,23 +1,43 @@
...
+#
+# For production it's recommended to configure a different adapter
+# at the `config/runtime.exs`.
+config :kepler, Kepler.Mailer, adapter: Swoosh.Adapters.Local
+
+# Swoosh API client is needed for adapters other than SMTP.
+config :swoosh, :api_client, false
+
+# Configure esbuild (the version is required)
+config :esbuild,
```
Comment (ashton314) [inline config/config.exs:33]: 😍

**Review by ashton314 — APPROVED** [review]
Two minor typographic edits and re-inclusion of the LICENSE; if you think I'm being too pedantic I'll politely disagree but I won't block a merge. ;-)

File: LICENSE (hunk 3 above)
Comment (alex0112) [reply LICENSE:1]: MIT is fine. Unless you wanted to go full GPLv3 someday haha.

**Pull request merged**